build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:0.12.2'
+        classpath 'com.android.tools.build:gradle:1.0.0'
```

Comment:
I don't understand these version numbers, as in: what should be correct? I find it strange that here it says 1.0.0, while in another file in this commit it changed from "gradle-1.12-all.zip" to "gradle-2.2.1-all.zip"

I guess it is correct though, just wondering